docs/source/contributing.md

```diff
@@ -0,0 +1,154 @@
+
+:::{important}
+We are in a bootstrapping phase to get an initial specification written.
+As part of this initial work we will propose a more formal governance model for the specification going forwards.
+
+We have chosen to keep all discussion to issues for now, so contributors have a single place to engage in conversation.
+Pull requests are used when a specific change is ready to be proposed.
+This can be without discussion, however, it is best for substantial or significant changes to be discussed first in an issue.
+We have opted to not use GitHub Discussions at this point.
+
+The community can suggest governance changes at any point.
+This includes the SATRE team, and any decision must be openly documented in the repo.
+:::
+
+### Specification Format
+
+The specification source is kept in the [specification repository](https://github.com/sa-tre/satre-specification).
+It is written as a [Sphinx](https://www.sphinx-doc.org/) document in [Markdown](https://www.markdownguide.org/) format.
+
+The most up-to-date 'source of truth' will be the specification on the `main` branch of the specification repository.
+The community can decide when to 'tag' a new version of the specification.
+They may also decide to where to publish the specification.
+
+The specification repository is self-contained and relates only to the specification specifically, or its governance.
+Any contributions to the wider SATRE project should be made through a different medium via the [SATRE GitHub organisation](https://github.com/sa-tre), or by contacting the SATRE team at [satre-contact@dundee.ac.uk](mailto:satre-contact@dundee.ac.uk).
+
+### Contribution Process
+
+Issues should be used to discuss ideas, potential changes and to ask questions.
+Issue templates have been designed for common issue archetypes to help collect the most important information and present it in a clear, consistent way.
+It is possible, however, to open a blank issue if none of the templates are suitable.
+
+When ready, changes will be proposed in pull requests.
+Similarly to issues, there is a pull request template.
+This template prompts contributors to include important details which helps explain the contribution and makes triage and review easier.
+
+Pull requests will be used to review changes.
+During the review process, the pull request will be used for discussion, to suggest amendments and ultimately accept or reject the change.
+
+We use this process to ensure that as much as possible of the discussion and decision making process can be public.
+This is to provide as open and accessible as possible an environment for all contributors to engage in the conversation.
+
+### Consensus Mechanism
+
+Approval from the SATRE team is based on [lazy consensus](https://medlabboulder.gitlab.io/democraticmediums/mediums/lazy_consensus/).
+Once a pull request has been open for 7 days, unless there are outstanding objections within the comments, the SATRE team will approve the change.
```

Comment:
```suggestion
Once a pull request has been open for at least 7 days, unless there are outstanding objections within the comments, the SATRE team may approve the change at any point.
```
Otherwise this implies it will be merged after 7 days, when we want more freedom to wait.

Reply:
Yes, I'm happy with any wording that doesn't imply we will merge immediately after 7 days.

Reply:
Updated in [4c4f5a4](https://github.com/sa-tre/satre-specification/pull/31/commits/4c4f5a466d6e9af9e595f9f02bbd128a507b1e61)